labgrid/target.py

```diff
@@ -244,9 +244,9 @@ def activate(self, client):
 
         # activate recursively and resolve conflicts
         for supplier in client.suppliers:
+            supplier.resolve_conflicts(client)
```

Comment:
I dropped this commit. I had it backwards and thought we wanted to deactivate the _suppliers_ not the clients.